website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -117,7 +117,7 @@ After initiating a graceful shutdown by either of these methods, the terminal us
 
 * `-name <name>`:
     An optional user-specified name for the agent. This name may be used in
-    the Terraform Cloud user interface to help easily identify the agent.
+    the HCP Terraform user interface to help easily identify the agent.
```

Comment:
All of the CLI docs are also auto-generated by the tfc-agent source code. Changing these here will be effectively moot, and upon the next release these will all be overwritten with the text from the tfc-agent's command line help output.

Reply:
Thank you for that context again; reverted the change! Same question as above, I'll assume your team is handling?